### DISCUSSION TITLE
Where is the promised "2021 file browser"?

### DISCUSSION THREAD

**Discussion by unixuser3** [ppy/osu#25692] [Ideas] [answered]
![image](https://github.com/ppy/osu/assets/78844597/c98953ff-52b2-46c0-b090-c6d580ef77ff)

Currently, everything in lazer files are encrypted under their SHA-256 hash. A way to browse these files was promised, and it was supposed to come online in 2021. It's 2023. Where is it?

**Comment by peppy** [comment, marked as answer]
You can export skins and beatmaps now, which should be lost if what you require.
